lib/tran_sock.h

```diff
@@ -125,7 +127,8 @@ vfu_msg(int sock, uint16_t msg_id,
         enum vfio_user_command cmd,
         void *send_data, size_t send_len,
         struct vfio_user_header *hdr,
-        void *recv_data, size_t recv_len);
+        void *recv_data, size_t recv_len,
```

Comment:
vfu_msg() is supposed to be the simpler version: please add a new vfu_msg_fds() that adds the new recv_fds.

Reply:
Good idea.